workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-search-category-facet-tabs/ddm-template.ftl

```diff
@@ -42,15 +42,30 @@
 </#if>
 
 <@liferay_aui.script>
```

Comment:
@luisacmn I am not sure I understand the bug. I do see a delay at times for the script to load, but I dont see the need for a refresh. I would like to avoid rewriting js logic whenever possible. If our problem is loading order of existing JS maybe we can debug that for this issue, instead of reimplementing the same existing logic over again